tests/plotting/test_plotting.py

```diff
@@ -1575,6 +1575,30 @@ def test_user_annotations_scalar_bar_volume(uniform):
     p.show(before_close_callback=verify_cache_image)
 
 
+@skip_no_plotting
+def test_scalar_bar_args_unmodifies_add_mesh(sphere):
+    sargs = {"vertical": True}
+    sargs_copy = sargs.copy()
+
+    p = pyvista.Plotter()
+    p.add_mesh(sphere, scalar_bar_args=sargs)
+    p.show()
+
+    assert sargs == sargs_copy
+
+
+@skip_no_plotting
+def test_scalar_bar_args_unmodifies_add_volume(uniform):
+    sargs = {"vertical": True}
+    sargs_copy = sargs.copy()
+
+    p = pyvista.Plotter()
+    p.add_volume(uniform, scalar_bar_args=sargs)
+    p.show()
+
+    assert sargs == sargs_copy
+
+
```

Comment:
Good idea adding these, thanks.

One bikeshed-level remark: the "unmodifies" in the name of the tests seems a bit off to me. It's as if the scalar bar args would be the ones doing the modification (or lack thereof). I agree with having the `scalar_bar_args` part first, so perhaps we could just change the `unmodifies` to `unmodified`, as in `test_scalar_bar_args_unmodified_add_mesh`? Again, not critical at all, just my impression. It's also fine if you want to leave it as is.